src/simulation/environment/eclipse/eclipse.cpp

```diff
@@ -119,7 +119,7 @@ void Eclipse::UpdateState(uint64_t CurrentSimNanos)
     for(scIt = this->scStateBuffer.begin(); scIt != this->scStateBuffer.end(); scIt++)
     {
         double tmpShadowFactor = 1.0; // 1.0 means 100% illumination (no eclipse)
-        double eclipsePlanetDistance = 0.0;
+        double eclipsePlanetDistance = 1e16;
```

Comment:
I don't like initializing this to 1e16.  In meters this is already close to the Pluto radius.  I would just leave it at zero and adjust the logic below?